From 756523f21b67ce9fa94f92b8d89116b59f453956 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@freescale.com>
Date: Mon, 21 Dec 2015 18:06:01 +0800
Subject: [PATCH 0787/5242] MLK-12047-1 ARM: dts: imx6sx-sabreauto.dts: add
 ADC support

commit  db0a640de1a93cf7a75cc0fadf424330bfd5b898 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add ADC support for imx6sx-sabreauto board.

Signed-off-by: Haibo Chen <haibo.chen@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sabreauto.dts |   18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-sabreauto.dts b/arch/arm/boot/dts/imx6sx-sabreauto.dts
index 3bb9564..9b876ab 100644
--- a/arch/arm/boot/dts/imx6sx-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6sx-sabreauto.dts
@@ -159,6 +159,14 @@
 		vin-supply = <&reg_can_en>;
 	};
 
+	reg_vref_3v3: regulator@4 {
+		compatible = "regulator-fixed";
+		regulator-name = "vref-3v3";
+		regulator-min-microvolt = <3300000>;
+		regulator-max-microvolt = <3300000>;
+	};
+
+
 	sound-cs42888 {
 		compatible = "fsl,imx6-sabreauto-cs42888",
 			   "fsl,imx-audio-cs42888";
@@ -187,6 +195,16 @@
 	};
 };
 
+&adc1 {
+	vref-supply = <&reg_vref_3v3>;
+	status = "okay";
+};
+
+&adc2 {
+	vref-supply = <&reg_vref_3v3>;
+	status = "okay";
+};
+
 &audmux {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_audmux_3>;
-- 
1.7.9.5

